website/pages/docs/commands/namespace/apply.mdx

```diff
@@ -42,3 +42,21 @@ Create a namespace with a quota
 $ nomad namespace apply -description "Prod API servers" -quota prod api-prod
 Successfully applied namespace "api-prod"!
 ```
+
+Remove a quota from a namespace
```

Comment:
```suggestion
## Remove a quota from a namespace
```

Reply:
I looked at a bunch of other docs pages and found it's most common not to h2 these, but rather list each example under a single h2. Examples header.

Reply:
Ah, this _is_ an example, not a separate section. Makes sense.